sdcm/tester.py

```diff
@@ -1654,17 +1654,19 @@ def run_stress(self, stress_cmd, duration=None):
     # pylint: disable=too-many-arguments,too-many-return-statements
     def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',
                           round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False,
-                          stop_test_on_failure=True):
+                          stop_test_on_failure=True, sla_role=None):
```

Comment:
I think we could use the credentials parameter, it's exactly what we need user/password

Reply:
removed